client/src/modules/templates/bhEmployeeSelect.tmpl.html

```diff
@@ -20,7 +20,7 @@
       on-select="$ctrl.onSelect($item)">
       <ui-select-match placeholder="{{ 'FORM.SELECT.EMPLOYEE' | translate }}"><span>{{$select.selected.display_name}}</span></ui-select-match>
       <ui-select-choices ui-select-focus-patch repeat="employee.uuid as employee in $ctrl.employees | filter: { 'display_name': $select.search }">
-        <span ng-bind-html="employee.display_name | highlight:$select.search"></span>
+        <span> ({{employee.reference}}) {{employee.display_name}}</span>
```

Comment:
This removes the highlighting.  It also doesn't filter on employee reference as you type.

Could you instead make an `hrLabel` similar to how the bhAccountSelect works?  For reference, [here it is](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/modules/templates/bhAccountSelect.tmpl.html#L25).

That way we preserve highlighting on the name and reference.

Reply:
Does this filter when you type EM.IMCK.XYZ or just on the employee name?  It looks like it is just on employee name.